chaincode/testtuple.go

```diff
@@ -72,8 +86,7 @@ func (testtuple *Testtuple) SetFromInput(db *LedgerDB, inp inputTesttuple) error
 			return err
 		}
 		dataManagerKey = inp.DataManagerKey
-		sort.Strings(dataSampleKeys)
-		testtuple.Certified = objectiveDataManagerKey == dataManagerKey && reflect.DeepEqual(objectiveDataSampleKeys, dataSampleKeys)
+		testtuple.Certified = objectiveDataManagerKey == dataManagerKey && isEqual(objectiveDataSampleKeys, dataSampleKeys)
```

Comment:
For future readers it would be helpful to explain (in a comment?) that the `isEqual` function ensures the order is the same (and not only that the elements are the same, possibly unsorted)

Reply:
Please ignore this comment, misunderstanding on my part